build/docker/alpine.Dockerfile

```diff
@@ -51,5 +51,27 @@ RUN apk --no-cache --update add \
 
 RUN dotnet --version
 
+RUN echo 'https://dl-cdn.alpinelinux.org/alpine/v3.14/community' >> /etc/apk/repositories
+RUN echo 'https://dl-cdn.alpinelinux.org/alpine/v3.14/main' >> /etc/apk/repositories
```

Comment:
remove, the packages exists in alpine 3.18 already (which is alpine:latest currently), https://pkgs.alpinelinux.org/packages?name=php82*&branch=v3.18&repo=&arch=&maintainer=